packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -191,27 +185,4 @@ const DumbKonnectorAccountTabs = props => {
   )
 }
 
-export const KonnectorAccountTabs = props => {
-  return (
-    <FlowProvider
-      initialTrigger={props.initialTrigger}
-      konnector={props.konnector}
-    >
-      {({ flow }) => <DumbKonnectorAccountTabs {...props} flow={flow} />}
-    </FlowProvider>
-  )
-}
-
-KonnectorAccountTabs.propTypes = {
-  konnector: PropTypes.object.isRequired,
-  initialTrigger: PropTypes.object.isRequired,
-  account: PropTypes.object.isRequired,
-  onAccountDeleted: PropTypes.func.isRequired,
-  addAccount: PropTypes.func.isRequired,
-  /** @type {string} Can be used to force the initial tab */
-  initialActiveTab: PropTypes.oneOf(['configuration', 'data']),
-  intentsApi: intentsApiProptype,
-  innerAccountModalOverrides: innerAccountModalOverridesProptype
-}
```

Comment:
why are we deleting the propTypes? Can we add them to `KonnectorAccountTabs` components?